streams/nl.m3u

```diff
@@ -127,39 +127,119 @@ https://video.uitzending.tv:19360/zo34/zo34.m3u8
+#EXTINF:-1 tvg-id="A1TV.nl" status="online",A1 TV (1080p) [Not 24/7]
 https://www.a1mediagroep.nl/hls/a1tv.m3u8
 #EXTINF:-1 tvg-id="" status="blocked",Radio NL (1080p) [Not 24/7]
 https://61ae39677a1ab.streamlock.net/radionl/radionl_1080p/playlist.m3u8?ref=Hoofdwebsite&token_endtime=1661322499&token_hash=yPCofcNBmbfuVCU4UqsLrQ6GgnKJGC5gQ0tn5_F9FSE=&token_starttime=0
-#EXTINF:-1 tvg-id="" status="online",XITE (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="XITE.nl" status="online",XITE (720p) [Not 24/7]
 https://ms2.mx-cd.net/tv/226-1855782/XITE_NL.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="online",XITE (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="XITE.nl" status="online",XITE (720p) [Not 24/7]
 https://ms7.mx-cd.net/tv/226-1855782/XITE_NL.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="timeout",Zed TV
 http://190.2.148.141:8080/bar7657/index.m3u8
+#EXTINF:-1 tvg-id="AltenaTV.nl",Altena TV (720p)
+https://dnx1f4sl30ba2.cloudfront.net/nlpo/clr-nlpo/altenatv/index.m3u8
+#EXTINF:-1 tvg-id="AAVISIE.nl",AAVISIE (720p)
+https://d1wgur45ikj8gz.cloudfront.net/nlpo/clr-nlpo/aavisietv/index.m3u8
+#EXTINF:-1 tvg-id="Regio90TV.nl",Regio90 TV (720p)
+https://ms2.mx-cd.net/dtv-02/260-2403096/Regio90TV.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="SirisTV.nl",Siris TV (720p)
+https://cdn.streammy.events/siris/playout/playlist.m3u8
+#EXTINF:-1 tvg-id="DTVOssBernheze.nl",DTV Oss & Bernheze (1080p)
+https://rrr.sz.xlcdn.com/?account=dtvmaasland&file=dtv1&type=live&service=wowza&protocol=https&output=playlist.m3u8
+#EXTINF:-1 tvg-id="DTVDenBosch.nl",DTV Den Bosch (1080p)
+https://rrr.sz.xlcdn.com/?account=dtvmaasland&file=dtv2&type=live&service=wowza&protocol=https&output=playlist.m3u8
+#EXTINF:-1 tvg-id="DTVMaashorst.nl",DTV Maashorst (1080p)
+https://rrr.sz.xlcdn.com/?account=dtvmaasland&file=dtv3&type=live&service=wowza&protocol=https&output=playlist.m3u8
+#EXTINF:-1 tvg-id="NOOSTV.nl",NOOS TV (720p)
+https://ms2.mx-cd.net/tv/276-2860564/Omroep_NOOS.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="1TwenteTV.nl",1Twente TV (1080p)
+https://ms2.mx-cd.net/dtv-10/198-989148/1Twente_TV.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="A1TV.nl",A1 TV (1080p) [Not 24/7]
+http://stream.a1mediagroep.nl/hls/a1tv.m3u8
+#EXTINF:-1 tvg-id="BEATTV.nl",BE@TTV (1080p)
+https://tv.beatfm.nl/live.m3u8
+#EXTINF:-1 tvg-id="HTRTV.nl",HTR TV (720p)
+https://ms7.mx-cd.net/tv/286-3043214/HTR_Media_TV.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="LangstraatTV.nl",Langstraat TV (720p)
+https://d1qpgfi3ivud4k.cloudfront.net/nlpo/clr-nlpo/langstraat/index.m3u8
+#EXTINF:-1 tvg-id="LEOMiddelseTV.nl",LEO Middelsé TV (480p)
+https://ms7.mx-cd.net/tv/238-2061506/Omroep_LEO_TV.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="LINQTV.nl",LINQ TV (480p)
+https://ms7.mx-cd.net/dtv-09/246-2090022/Linq_Media_TV.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="AssenTV.nl",Assen TV (720p)
+https://n1.cdn.jetstre.am/session/a179f3e35120dedc2c1ce303188a0be4/sz/OmroepAssen/webdav/live/OmroepAssenTV/playlist.m3u8?token=NwN4jakl9bcUTrZdSIVRVQ&time=1675605989
```

Comment:
This one is tokened.

Reply:
though I have another one : RTV Drenthe https://cdn.rtvdrenthe.nl/live/rtvdrenthe/tv/index.m3u8

Reply:
Radio is https://rrr.sz.xlcdn.com/?account=OmroepAssen&file=RadioOmroepAssen&type=live&service=icecast&protocol=https&port=8000&output=pls
TV is https://takeoff.jetstre.am/?account=OmroepAssen&file=OmroepAssenTV%2Fplaylist.m3u8&type=live&service=webdav&protocol=https&output=playlist.m3u8

Reply:
> This one is tokened.

Got the links from a netherlands forum. Link still work since 1 year ago even with token so idk lol

Reply:
Rather odd then, doesn't seem to work on my side.